packaging/examples/metrics/grafana-dashboards/strimzi-kafka.json

```diff
@@ -960,7 +960,7 @@
       "targets": [
         {
           "datasource": "${DS_PROMETHEUS}",
-          "expr": "sum(kubelet_volume_stats_available_bytes{namespace=\"$kubernetes_namespace\",persistentvolumeclaim=~\"data(-[0-9]+)?-$strimzi_cluster_name-(kafka|$pool_name)-[0-9]+\"}) by (persistentvolumeclaim)",
+          "expr": "sum(kubelet_volume_stats_available_bytes{namespace=\"$kubernetes_namespace\",persistentvolumeclaim=~\"data(-[0-9]+)?-$strimzi_cluster_name-$kafka_broker\"}) by (persistentvolumeclaim)",
```

Comment:
How does this work with node pools as you are replacing the pool name?

Also, I guess this change should be done to the KRaft dashboard as well?

Reply:
The `$kafka_broker` will change based on the pool/node. 
![strimzi-kafka](https://github.com/strimzi/strimzi-kafka-operator/assets/56991288/2eeaefe3-8c5b-4d2e-871f-c74543646e54)

Reply:
But it defaults to something like `.*` => so that surely doesn't work the same way as what was there. It should be now matching all PVCs which seems more wrong then ignoring the broker selector. Or how does it filter out non-Kafka PVCs?

Reply:
> But it defaults to something like `.*` => so that surely doesn't work the same way as what was there. It should be now matching all PVCs which seems more wrong then ignoring the broker selector. Or how does it filter out non-Kafka PVCs?

I am not really understand what you mean. The current filter should works. It only filter the pvcs which has name: `data(-[0-9]+)?-$strimzi_cluster_name-$kafka_broker`

<img width="1516" alt="image" src="https://github.com/strimzi/strimzi-kafka-operator/assets/56991288/9e52a838-1254-4df2-8bb5-71d0dbed31e4">

Reply:
How about this?
`kubelet_volume_stats_available_bytes{namespace="multicluster-global-hub",persistentvolumeclaim=~"data(-[0-9]+)?-my-cluster-.*",persistentvolumeclaim!~".*zookeeper(-[0-9]+)?$"}`

Reply:
I'm not sure if that would work in all environments. If yes, it might be an option I guess.

Reply:
Changed the code to use `kubelet_volume_stats_available_bytes{namespace="multicluster-global-hub",persistentvolumeclaim=~"data(-[0-9]+)?-my-cluster-.*",persistentvolumeclaim!~".*zookeeper(-[0-9]+)?$"}` please help to review again.